components/FeaturesSection.tsx

```diff
@@ -97,44 +97,89 @@ const FeaturesSection = () => {
-              />
-              <FeatureCard
-                image={subdomainWindow}
-                title={'Live collaboration '}
-                text="Create a document or Whiteboard directly on Fileverse Portal and collaborate live with your peers. All stored on IPFS!"
-              />
-            </>
+                />
+                <FeatureCard
+                  image={chatWindow}
+                  title={'Chat Rooms & Spaces'}
+                  text=" Create encrypted chat rooms to coordinate. Invite your peers by sending access-NFTs. Use your ENS as your chat profile."
+                />
+                <FeatureCard
+                  image={threeDbeauty}
+                  title={'3D Beauty '}
+                  text="Upload your 3D files and share the URL with anons, frens, or the world. View any 3D model directly on Fileverse."
+                />
+              </>
+            ) : (
+              <>
+                <FeatureCard
+                  image={dPages}
+                  title={'Decentralized Pages'}
+                  text="Your Notion alternative - publish decentralized websites, blogs, technical docs, write your notes or create personal wikis/databases, and more."
+                />
+                <FeatureCard
+                  image={accessToken}
+                  title={'Airdrop Access-Tokens'}
+                  text="These tokens are NFTs that you can fully customise and send to as many addresses you want to give them access to files.
+                "
+                />
+                <FeatureCard
+                  image={threeDbeauty}
+                  title={'3D Beauty '}
+                  text="Upload your 3D files and share the URL with anons, frens, or the world. View any 3D model directly on Fileverse."
+                />
+                <FeatureCard
+                  image={tokenGating}
+                  title={'Token Gating'}
+                  text={
+                    'Create granular permissions for individuals or communities accessing your files and content. All via onchain rules.'
+                  }
+                />
+                <FeatureCard
+                  image={fileTypes}
+                  title={'Multimedia'}
+                  text=" Upload and share any type of multimedia file. Your peers will be able to preview or download them directly on Fileverse."
+                />
+                <FeatureCard
+                  image={subdomainWindow}
+                  title={'Live collaboration '}
+                  text="Create a doc, whiteboard, or web page directly on Fileverse and collaborate with your peers. Stored on IPFS/Arweave!"
+                />
+                <FeatureCard
+                  image={chatWindow}
+                  title={'Chat Rooms & Spaces'}
+                  text=" Create encrypted chat rooms to coordinate. Invite your peers by sending access-NFTs. Use your ENS as your chat profile."
+                />
+              </>
```

Comment:
Please consider put all props into a constant array like const FEATURES = [], then iterate the array length with the FeatureCard component. This will make your code cleaner, shorter and easy to maintain.

Reply:
done